lib/src/main/java/com/what3words/components/maps/wrappers/W3WMapBoxWrapper.kt

```diff
@@ -807,40 +837,96 @@ class W3WMapBoxWrapper(
         }
     }
 
-    /** [drawCircle] will be responsible for the drawing of the w3w circle pins (i.e [R.drawable.ic_marker_circle_red]) by adding them to [MapboxMap.style] using a [SymbolLayer].*/
-    private fun drawCircle(data: SuggestionWithCoordinatesAndStyle) {
-        data.markerColor.toCircle().let { markerFillColor ->
-            bitmapFromDrawableRes(context, markerFillColor)?.let { bitmap ->
-                mapView.getStyle { style ->
-                    style.addImage(
-                        String.format(CIRCLE_ID_PREFIX, data.markerColor.toString()),
-                        bitmap
-                    )
-                    if (!style.styleSourceExists(data.suggestion.words)) {
-                        style.addSource(
-                            geoJsonSource(data.suggestion.words) {
-                                geometry(
-                                    Point.fromLngLat(
-                                        data.suggestion.coordinates.lng,
-                                        data.suggestion.coordinates.lat
-                                    )
-                                )
-                            }
+    /** [drawPin] will be responsible for the drawing of the selected and added w3w pins (i.e [R.drawable.ic_marker_pin_red]) by adding them to [MapboxMap.style] using a [SymbolLayer].*/
+    private fun drawPin(data: W3WAddressWithStyle) {
+        data.address.center?.let { center ->
```

Comment:
Maybe I'm wrong but in W3WGoogleMapWrapper, we have a function to draw the pin at the word when the coordinates are missing, but I don't see a similar function here.